src/Microsoft.Health.Fhir.Tests.Common/TestFiles/Normative/SearchParameter-AppointmentStatus.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
I like how the name of the file indicates what the search parameter is used for. It might be nice to continue this pattern and update the existing `SearchParameter.json` file to `SearchParameter-PatientName.json`.